pkg/proxy/iptables/proxier.go

```diff
@@ -778,10 +901,22 @@ func (proxier *Proxier) syncProxyRules() {
 		}
 		activeNATChains[svcChain] = true
 
+		svcLbChain := serviceLBChainName(svcName, protocol)
+		if svcInfo.onlyNodeLocalEndpoints {
+			// Create the per-service LB chain, retaining counters if possible.
+			// only for ESIPP annotation services
+			if lbChain, ok := existingNATChains[svcLbChain]; ok {
+				writeLine(natChains, lbChain)
+			} else {
+				writeLine(natChains, utiliptables.MakeChainLine(svcLbChain))
+			}
+			activeNATChains[svcLbChain] = true
+		}
+
 		// Capture the clusterIP.
 		args := []string{
 			"-A", string(kubeServicesChain),
-			"-m", "comment", "--comment", fmt.Sprintf(`"%s cluster IP"`, svcName.String()),
+			"-m", "comment", "--comment", fmt.Sprintf(`"%s cluster IP (svc-guid: %s)"`, svcName.String(), svcInfo.guid),
```

Comment:
why are you sticking UID in the comment?  Is it to further uniquify or just for more information?  I don't think we need the extra information...